src/megaclient.cpp

```diff
@@ -7876,6 +7962,12 @@ bool MegaClient::fetchsc(DbTable* sctable)
 
     while (sctable->next(&id, &data, &key))
     {
+        if (fnstats.timeToFirstByte == NEVER)
```

Comment:
Maybe we should move the assignment of `timeToFirstByte` out of the loop, since it will iterate thousands of times and only once the condition will be true.